DESCRIPTION

```diff
@@ -110,10 +110,14 @@ Config/Needs/website:
     tidyr
 Config/Needs/deploy:
     BH,
+    dplyr,
     DT,
+    GGally,
```

Comment:
I think it's worth using `rlang::check_installed("GGally")` for this dependency in the theme preview app so that users who don't have this package installed are prompted to install it.

Reply:
I do like this for packages that are needed in a "global" sense, but for something like `ggridges`, which is just needed for a singular output, I'd rather let the usual error be shown in the app (if you call `check_installed()` in a reactive, you just get a blank UI, which could be confusing).

Reply:
Totally. I only meant this comment for GGally which was a bigger component